src/components/TodoList.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+import { Todos } from './Todos';
+
+interface Props {
+  preparedTodos: Todo[];
+  setTodos: (value: Todo[]) => void;
+  handleError: (value: string) => void;
+  tempTodo: Todo | null;
+  focusInput: () => void;
+}
+
+export const TodoList: React.FC<Props> = ({
+  preparedTodos,
+  setTodos,
+  handleError,
+  tempTodo,
+  focusInput,
+}) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      <Todos
+        preparedTodos={preparedTodos}
+        setTodos={setTodos}
+        handleError={handleError}
+        tempTodo={tempTodo}
+        focusInput={focusInput}
+      />
```

Comment:
In the future, it'll be much easier to create a `Context` instead of drilling props into components